### PR TITLE
chore(TUP-29079): remove plugins to fix cve

### PR DESCRIPTION
**Pull request by hzhao-talendbj**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.emr450/plugin.xml [hunk 1]
```diff
@@ -269,8 +269,7 @@
             context="plugin:org.talend.libraries.zookeeper"
             id="zookeeper-emr450-latest"
             name="zookeeper-3.4.8.jar" 
-            mvn_uri="mvn:org.talend.libraries/zookeeper-3.4.8/6.0.0"
-            uripath="platform:/plugin/org.talend.libraries.zookeeper/lib/zookeeper-3.4.8.jar"/>
+            mvn_uri="mvn:org.talend.libraries/zookeeper-3.4.8/6.0.0"/>
```
Comment (bhe-talendbj) [inline main/plugins/org.talend.hadoop.distribution.emr450/plugin.xml:272]: Why not maven_uri = "mvn:org.apache.zookeeper/zookeeper/3.4.8"? I suggest to change value of all maven uri to official uri by default.

Comment (lbourgeois) [reply main/plugins/org.talend.hadoop.distribution.emr450/plugin.xml:272]: That's what we are targetting for new jars. If we change existing jars we will need to upload it to new location

Comment (hzhao-talendbj) [reply main/plugins/org.talend.hadoop.distribution.emr450/plugin.xml:272]: @lbourgeois  hello 
we uploaded the jar to talend-update,  so the jar will be asked to download from nexus

Comment (lbourgeois) [reply main/plugins/org.talend.hadoop.distribution.emr450/plugin.xml:272]: Correct me if I am wrong but it was already the case (donwloading from Nexus) as long as mvn_uri is provided uripath is useless

**Review by lbourgeois — APPROVED** [review]
LGTM but same lib versions are still referenced so  CVE alerts will persist right ?

File: main/plugins/org.talend.hadoop.distribution.emr450/plugin.xml (hunk 1 above)
Comment (hzhao-talendbj) [reply main/plugins/org.talend.hadoop.distribution.emr450/plugin.xml:272]: yes ,you are right .the cve scan system is only scan jars based on jar inside studio or not. so here we only removed the jars from studio. maybe later there will be tasks for BD side to upgrade the cve jars.

Comment (hzhao-talendbj) [reply main/plugins/org.talend.hadoop.distribution.emr450/plugin.xml:272]: studio will try to find the jar based on uripath first , then mvn_uri.  so might save few time on the check of studio if we remove the uripath

**Review by zwxue — APPROVED** [review]

**Pull request merged**